Tests/W3CTraceContextTests/TraceParentTests.swift

```diff
@@ -150,6 +150,15 @@ final class TraceParentRawRepresentableTests: XCTestCase {
 
         XCTAssertNotEqual(parent1, parent2)
     }
+
+    // MARK: - Random
+
+    func test_generate_random_traceParent() {
+        let traceParent = TraceParent.random()
+
+        // validate random trace-parent by parsing its raw value
+        XCTAssertNotNil(TraceParent(rawValue: traceParent.rawValue))
```

Comment:
should it rather be equal to `traceParent` rather than just not-nil?

Reply:
Yep. Good catch, will update! 👌